.github/scripts/test_tekton_tasks.sh

```diff
@@ -116,7 +136,13 @@ do
       sleep 5
     done
 
-    PIPELINERUN=$(tkn p start $TEST_NAME -w name=tests-workspace,volumeClaimTemplateFile=$WORKSPACE_TEMPLATE -o json | jq -r '.metadata.name')
+    if [ -z "${USE_TRUSTED_ARTIFACTS}" ]; then
+      workSpaceParams="volumeClaimTemplateFile=$WORKSPACE_TEMPLATE"
+    else
+      workSpaceParams="emptyDir="""
```

Comment:
I don't think the `""` at the end make a difference. When bash evaluates this, it will just see one string `emptyDir=` plus an empty string which together is still just `emptyDir=`. So:
```suggestion
      workSpaceParams="emptyDir="
```

Reply:
fixed